src/components/loadflow-result.js

```diff
@@ -241,6 +254,47 @@ const LoadFlowResult = ({ result, studyUuid, nodeUuid }) => {
         ];
     }, [intl, NumberRenderer, StatusCellRender]);
 
+    const renderLimitType = useCallback(
+        (limitType) => {
+            switch (limitType) {
+                case LIMIT_TYPES.HIGH_VOLTAGE:
+                    return intl.formatMessage({ id: 'HIGH_VOLTAGE' });
+                case LIMIT_TYPES.LOW_VOLTAGE:
+                    return intl.formatMessage({ id: 'LOW_VOLTAGE' });
+                case LIMIT_TYPES.CURRENT:
+                    return intl.formatMessage({ id: 'CURRENT' });
+                default:
+                    return limitType;
+            }
+        },
+        [intl, LIMIT_TYPES]
+    );
+    const loadFlowVoltageViolationscolumns = useMemo(() => {
+        return [
+            {
+                headerName: intl.formatMessage({ id: 'VoltageLevel' }),
+                field: 'name',
+                numeric: false,
```

Comment:
I think it's the default value, we don't need to specify it

Reply:
Done